parts/k8s/kubernetesconfigs.sh

```diff
@@ -149,6 +149,7 @@ function configureK8s() {
     "cloudProviderRateLimitQPS": ${CLOUDPROVIDER_RATELIMIT_QPS},
     "cloudProviderRateLimitBucket": ${CLOUDPROVIDER_RATELIMIT_BUCKET},
     "useManagedIdentityExtension": ${USE_MANAGED_IDENTITY_EXTENSION},
+    "userAssignedIdentityID": "${USE_ASSIGNED_IDENTITY_ID}",
```

Comment:
`s/{USE_ASSIGNED_IDENTITY_ID}/{USER_ASSIGNED_IDENTITY_ID}` (note the add'l "R" in `USER`)